tsl/test/sql/transparent_decompression_ordered_index.sql.in

```diff
@@ -0,0 +1,132 @@
+-- This file and its contents are licensed under the Timescale License.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-TIMESCALE for a copy of the license.
+
+\set TEST_BASE_NAME transparent_decompression
+SELECT format('include/%s_query.sql', :'TEST_BASE_NAME') AS "TEST_QUERY_NAME",
+    format('%s/results/%s_results_uncompressed.out', :'TEST_OUTPUT_DIR', :'TEST_BASE_NAME') AS "TEST_RESULTS_UNCOMPRESSED",
+    format('%s/results/%s_results_compressed.out', :'TEST_OUTPUT_DIR', :'TEST_BASE_NAME') AS "TEST_RESULTS_COMPRESSED" \gset
+
+SELECT format('\! diff %s %s', :'TEST_RESULTS_UNCOMPRESSED', :'TEST_RESULTS_COMPRESSED') AS "DIFF_CMD" \gset
+
+-- Testing Index Scan backwards ----
+--want more than 1 segment in atleast 1 of the chunks
+
+CREATE TABLE metrics_ordered_idx (
+    time timestamptz NOT NULL,
+    device_id int,
+    device_id_peer int,
+    v0 int
+);
+
+SELECT create_hypertable ('metrics_ordered_idx', 'time', chunk_time_interval => '2days'::interval);
+
+ALTER TABLE metrics_ordered_idx SET (timescaledb.compress, timescaledb.compress_orderby = 'time ASC', timescaledb.compress_segmentby = 'device_id,device_id_peer');
+
+INSERT INTO metrics_ordered_idx (time, device_id, device_id_peer, v0)
+SELECT time,
+    device_id,
+    0,
+    device_id
+FROM generate_series('2000-01-13 0:00:00+0'::timestamptz, '2000-01-15 23:55:00+0', '15m') gtime (time),
```

Comment:
This table now has data for every 15m instead of 5m. Does it still satisfy the requirement of having atleast 1 chunk with > = 2 segments?

Reply:
One of the problems with the way the compression tests are organized is that we cannot really figure out if the query output has changed since this only diffs output between compressed and uncompressed versions. I  have seen tests where the query outputs 0 rows.

Reply:
you can see those in explain output as it has counts for the tuples, it might be not superobvious but its there. i don't think adding output adds much value here thats just a huge amount of data. thats why we have uncompressed hypertables with the same data so we can diff the results.

Reply:
It helps for cases like these when you can see how the output changes due to the data set change.  Otherwise, the developer would presume changes are fine since we don't see any diffs while running the tests.